cmake/support/AC_Utils.cmake

```diff
@@ -98,7 +98,7 @@ function(acwrap AC_TYPE AC_FINAL_SOURCE AC_FINAL_HEADER AI_XML XML_FILE_DEPS MOD
       COMMAND ${CMAKE_COMMAND} -E chdir ${CMAKE_CURRENT_SOURCE_DIR}
       ${CMAKE_COMMAND} -E env PYTHONPATH=${PYTHON_AUTOCODER_DIR}/src:${PYTHON_AUTOCODER_DIR}/utils BUILD_ROOT="${FPRIME_BUILD_LOCATIONS_SEP}"
       FPRIME_AC_CONSTANTS_FILE="${FPRIME_AC_CONSTANTS_FILE}"
-      PYTHON_AUTOCODER_DIR=${PYTHON_AUTOCODER_DIR} DICTIONARY_DIR=${DICTIONARY_DIR}
+      PYTHON_AUTOCODER_DIR=${PYTHON_AUTOCODER_DIR}
```

Comment:
Why did you remove DICTIONARY_DIR here?

Reply:
DICTIONARY_DIR is never set, so it makes the dictionary target logic less confusing to have it removed. I think I removed it everywhere but DICTIONARY_DIR got removed in other changes, so after rebasing this was the only spot left, making it seem a little random.